go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rancher/shepherd
 
-go 1.22
+go 1.22.0
```

Comment:
why is this change necessary?

Reply:
It is only necessary on macs due to a golang issue. I will remove it.